src/ert/gui/simulation/run_dialog.py

```diff
@@ -230,7 +230,7 @@ def _select_real(self, index):
         iter_ = index.model().get_iter()
         self._job_model.set_real(iter_, real)
         self._job_label.setText(
-            f"Realization id {index.data(RealIens)} in iteration {iter_}"
+            f"Realization id {index.data(RealIens)} in iteration {index.data(IterNum)}"
```

Comment:
This might be correct, but the `iter_` that was earlier used in the text, is also used/passed to the set_real-function. Please verify that this is the intended behaviour.

Reply:
I think the iteration will still be mapped to `0` in the underlying data. For instance the `.get_iter()` function seems to return the index of the rowsInserted into the snapshot model, and I guess this will always start at `0`. I guess then that the only thing to do is to display the correct iteration id?

Reply:
I think `iter` is somewhat "misused" in the GUI components, as it is there used to keep track of the indicies of for instance the RealizationWidget. Since we only run one iteration then this index must be `0` and not the real iteration number.

Reply:
I see this function leads to a signal, `self.real_changed.emit(iter_, real)`.
I can't immediately see that this signal is caught anywhere, can you look into that?

Reply:
I cant find any receivers either. Tried to check by running `self.receivers(self.real_changed)` which returns `0`. Not entirely sure if this is reliable though.

Reply:
And cant see any obvious difference when uncommenting the emit line.